sprinklers/base.py

```diff
@@ -51,9 +76,18 @@ def start(self):
         qs = self.get_queryset()
         ids = [o['id'] if isinstance(o, dict) else o.id for o in qs]
 
+        async_subtask = (
+            self._async_subtask
+            if isinstance(getattr(self, '_async_subtask', None), Task)
+            else _async_subtask
+        )
```

Comment:
This was totally @stephen-bunn's crazy idea, FWIW. Blame him.

Reply:
Does this actually work? Celery needs to be able to import the method for the task